puppet/modules/jenkins_job_builder/files/centos.org/ansible/provision-duffy-test.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This playbook could use a name a bit more closer to what it actually does I feel.